docs/gdd.rst

```diff
@@ -130,6 +130,27 @@ Some effects (e.g., damage) make use of a derived "strength" stat.
 This is the ability power (AP) multiplied by one of physical attack (PA) or magical attack (MA) depending on the ability type.
 For example the damage of a magical attack would be ``AP * MA``.
 
+Job Point (JP) formula
+	Current: The running total of the power so far.
```

Comment:
I don't see "Current" used/reference elsewhere.

Reply:
I had removed it, in exchange for the algorithm at the bottom, but forgot to remove this.